src/internal/converters/eml/eml.go

```diff
@@ -0,0 +1,142 @@
+package eml
+
+// This package helps convert from the json response
+// received from Graph API to .eml format (rfc0822).
+// Ref: https://www.ietf.org/rfc/rfc0822.txt
+// Ref: https://datatracker.ietf.org/doc/html/rfc5322
+// Data missing from backup:
+// SetReturnPath SetPriority SetListUnsubscribe SetDkim
+// AddAlternative SetDSN (and any other X-MS specific headers)
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/alcionai/clues"
+	"github.com/microsoftgraph/msgraph-sdk-go/models"
+	mail "github.com/xhit/go-simple-mail/v2"
+
+	"github.com/alcionai/corso/src/internal/common/ptr"
+	"github.com/alcionai/corso/src/pkg/logger"
+)
+
+const (
+	fromFormat = "%s <%s>"
+	dateFormat = "2006-01-02 15:04:05 MST" // from xhit/go-simple-mail
+)
+
+// toEml converts a Messageable to .eml format
+func toEml(data models.Messageable) (string, error) {
+	email := mail.NewMSG()
+
+	if data.GetFrom() != nil {
+		email.SetFrom(
+			fmt.Sprintf(
+				fromFormat,
+				ptr.Val(data.GetFrom().GetEmailAddress().GetName()),
+				ptr.Val(data.GetFrom().GetEmailAddress().GetAddress())))
```

Comment:
this applies to the other times we use this pattern as well, but do we need to format things differently if the name is empty? If we feel like this is something we're going to be tweaking in the future maybe it's worth making a function that goes from the result of `GetEmailAddress()` to a string ready to be inserted into the eml library? That would help ensure it stays consistent across fields

Reply:
Fixed in https://github.com/alcionai/corso/pull/4651